docs/portal/registry-users/viewing-users.md

```diff
@@ -0,0 +1,16 @@
+# Viewing Registry Users
+
+The following procedure describes how to view information about registry users. Buttons at the top right of the displayed form allow you to view logs associated with the registry user, delete the registry user, and update the registry user.
+
+1. In the left pane, click **Image Registry > Registry Users**.
+2. Click the name of the registry user whose information you want to view. A form similar to the following appears. The table below the figure describes the elements on this form.
+
+    ![null](</docs/resources/images/registry/edit-user.png>)
+
+| **Element Number**       | **Description**                               |
+| -------------------------|-----------------------------------------------| 
+1                                                                                                       | The Basic Information area shows basic information about the registry user. To edit a project associated with the registry user, click a project in the table within the [Access to Projects field](</docs/portal/image-registry/viewing-projects.md>).                                            |
```

Comment:
Please change the following:
Change "To edit a project" to "To view a project"
Place the link to "viewing projects page" behind the new "view a project" instead of "Access to Projects field"

Reply:
Done.